lang/main-nl.json

```diff
@@ -497,6 +497,7 @@
         "errorMissingPassword": "Voer alstublieft het wachtwoord van de vergadering in",
         "invalidPassword": "Ongeldig wachtwoord",
         "joinRejectedMessage": "Uw verzoek tot deelname is afgewezen door een moderator.",
+        "joinRejectedTitle": "Verzoek tot deelname afgewezen.",
```

Comment:
I think this shouldn't end in a period

Reply:
See main.json, there this string also ends with a period.
Should I remove this period?

Reply:
Oh, yeah please do!

Reply:
Removed!